### PR TITLE
[PORT] removes bloodsucker objectives

### PR DESCRIPTION
**Pull request by Moltijoe**
not quite full port
https://github.com/fulpstation/fulpstation/pull/1049

# Why is this good for the game?
The objectives aren't that interesting and really just force bloodsuckers to act weirdly entirely for greentext, by nature bloodsuckers end up being on the louder end so them existing at all is usually enough to drive a round.

clan specific objectives still remain

keeps the vassal objectives by request

:cl:  @JohnFulpWillard @Moltijoe 
rscdel: Bloodsuckers can no longer get an objective to either drink blood or collect hearts
/:cl:

### CODE REVIEW

**Comment by solwashere** [comment]
Fine with removing drinking blood, but vassilizing is one of the few non murdery objectives that we have for an antag

**Comment by Moltijoe** [comment]
> Fine with removing drinking blood, but vassilizing is one of the few non murdery objectives that we have for an antag

clans still have their clan specific objective
this means they'll still have other things to do, just not things that are basically the tutorial for the antag

also vassalization is just the same as other conversion antags
bloodsucker shouldn't be forced into being a conversion antag unless it fits their clan

**Comment by neeshacark** [comment]
Speaking from a staff perspective I feel like removing these objectives is going to cause a lot of ahelps for 'hey can i get some objectives' because people don't like not having directives to follow (they cant think of their own gimmick)

If we remove them they need to be replaced by something, either the new gimmick objectives or the generic objectives we use for traitors and such.

**Comment by Moltijoe** [comment]
> Speaking from a staff perspective I feel like removing these objectives is going to cause a lot of ahelps for 'hey can i get some objectives' because people don't like not having directives to follow (they cant think of their own gimmick)
> 
> If we remove them they need to be replaced by something, either the new gimmick objectives or the generic objectives we use for traitors and such.

they still have a vassal target, if they manage to get that done as well as their clan objective and STILL want more objectives, that's them being an overachiever

File: code/modules/antagonists/bloodsuckers/clans/clan_gangrel.dm [hunk 1]
```diff
@@ -21,3 +21,23 @@
 /datum/bloodsucker_clan/malkavian/on_favorite_vassal(datum/antagonist/bloodsucker/source, datum/antagonist/vassal/vassaldatum)
```
Comment (tattax) [inline code/modules/antagonists/bloodsuckers/clans/clan_gangrel.dm]:
```suggestion
/datum/bloodsucker_clan/gangrel/on_favorite_vassal(datum/antagonist/bloodsucker/source, datum/antagonist/vassal/vassaldatum)
```

File: code/modules/antagonists/bloodsuckers/clans/clan_gangrel.dm [hunk 2]
```diff
@@ -21,3 +21,23 @@
...
 	var/datum/action/cooldown/spell/shapeshift/bat/batform = new(vassaldatum.owner || vassaldatum.owner.current)
 	batform.Grant(vassaldatum.owner.current)
+
+/// Enter Frenzy repeatedly
+/datum/objective/bloodsucker/frenzy
```
Comment (tattax) [inline code/modules/antagonists/bloodsuckers/clans/clan_gangrel.dm]: path

File: code/modules/antagonists/bloodsuckers/clans/clan_gangrel.dm (hunk 1 above)
Comment (Moltijoe) [reply code/modules/antagonists/bloodsuckers/clans/clan_gangrel.dm]: lol, that was tattax apparently

File: code/modules/antagonists/bloodsuckers/clans/clan_gangrel.dm (hunk 2 above)
Comment (Moltijoe) [reply code/modules/antagonists/bloodsuckers/clans/clan_gangrel.dm]: ty

**Review by ToasterBiome — APPROVED** [review]

**Pull request merged**